Sources/ComposableArchitecture/SwiftUI/IfLetStore.swift

```diff
@@ -10,25 +10,27 @@ import SwiftUI
 /// ```swift
 /// IfLetStore(
 ///   store.scope(state: \SearchState.results, action: SearchAction.results),
-///   then: SearchResultsView.init(store:),
-///   else: { Text("Loading search results...") }
-/// )
+/// ) {
+///   SearchResultsView(store: $0)
+/// } else {
```

Comment:
Typo, should be `else:`.

Reply:
@gohanlon good eye! fixed here b66fe3d7ea7a4e16f0a0942f15c246fcf11be094